src/data/markdown/docs/02 javascript api/07 k6-experimental/02 grpc/20-Params.md

```diff
@@ -8,7 +8,7 @@ excerpt: 'Params is an object used by the gRPC methods that generate RPC request
 
 | Name | Type | Description |
 |------|------|-------------|
-| `Params.metadata` | object | Object with key-value pairs representing custom metadata the user would like to add to the request. |
+| `Params.metadata` | object | Object with key-value pairs representing custom metadata the user would like to add to the request. Values of [keys ending with `-bin`]((https://grpc.io/docs/what-is-grpc/core-concepts/#metadata)) will be treated as binary data. |
```

Comment:
```suggestion
| `Params.metadata` | object | Object with key-value pairs representing custom metadata the user would like to add to the request. Values of [keys ending with `-bin`](https://grpc.io/docs/what-is-grpc/core-concepts/#metadata) will be treated as binary data. |
```

This is being displayed correctly in the preview, but is there an extra set of parenthesis here?

Reply:
Thanks, fixed in both places :+1: